ifrs17/Report/ParameterReportsQueries.ipynb

```diff
@@ -0,0 +1,400 @@
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public record SingleDataNodeReportParameter : ReportParameter  {",
+                "\n    public double PremiumAllocation { get; init; }",
+                "\n}"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public record InterDataNodeReportParameter : ReportParameter  {",
+                "\n    [Dimension(typeof(GroupOfContract), nameof(LinkedDataNode))]",
+                "\n    public string LinkedDataNode { get; init; }",
+                "\n",
+                "\n    public double ReinsuranceCoverage { get; init; }",
+                "\n}"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public record PartnerRatingsReportParameter : ReportParameter  {",
+                "\n    [IdentityProperty]",
+                "\n    [NotVisible]",
+                "\n    [Dimension(typeof(Partner))]",
+                "\n    public string Partner { get; init; }",
+                "\n",
+                "\n    [Dimension(typeof(CreditRiskRating))]",
+                "\n    public string CreditRiskRating { get; init; }",
+                "\n}",
+                "\npublic record CreditDefaultRatesReportParameter : ReportParameter  {",
+                "\n    [IdentityProperty]",
+                "\n    [Dimension(typeof(CreditRiskRating))]",
+                "\n    [NotVisible]",
+                "\n    public string CreditRiskRating { get; init; }",
+                "\n}"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Data Node"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public static async Task<DataNodeData[]> GetDataNodeDataReportParameters(this IWorkspace workspace, ImportArgs args) =>",
+                "\n    (await workspace.LoadDataNodesAsync(args))",
+                "\n    .Values",
+                "\n    .ToArray();"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public static async Task<DataNodeStateReportParameter[]> GetDataNodeStateReportParameters(this IWorkspace workspace, ImportArgs args) =>",
+                "\n    (await workspace.LoadCurrentAndPreviousParameterAsync<DataNodeState>(args, x => x.DataNode))",
```

Comment:
I was checking this query I don't see where args.ReportingNode is used in the filter. I think the query expects the workspace to have a  partition set. 
We can improve this in a following PR.